src/text2png.rs

```diff
@@ -0,0 +1,182 @@
+use harfrust::FontRef;
```

Comment:
I think this is copied from somewhere, can you add a note indicating the source?

//! renders text into pngs, originally copied from ...

Reply:
Done